doc/index.rst

```diff
@@ -116,4 +116,26 @@ NEST is available under the :doc:`GNU General Public License 2 or later <license
 -  modify and improve NEST according to your needs,
 -  distribute NEST to others under the same license.
 
-
+Acknowledgements
+################
+
+This project has received funding from the European Union’s Horizon 2020 Framework Programme for Research and
+Innovation under Specific Grant Agreement No. 945539 (Human Brain Project SGA3).
+
+The authors gratefully acknowledge the received support and funding from the European Union 6th and 7th Framework
+Program under grant agreement no. 15879 (FACETS), the European Union 7th Framework Program under grant agreement no.
+269921 (BrainScaleS), the European Union 7th Framework Programme ([FP7/2007-2013]) under grant agreement no. 604102
+(Human Brain Project, HBP), the computing time granted by the JARA-HPC Vergabegremium and provided on the JARA-HPC
+Partition part of the supercomputers JUQUEEN and JURECA at Forschungszentrum Jülich (VSR computation time grant
+JINB33), the Jülich Aachen Research Alliance (JARA), the Next-Generation Supercomputer Project of MEXT, Japan, the
+eScience program of the Research Council of Norway under grant 178892/V30 (eNeuro), the Helmholtz Association through
+the Helmholtz Portfolio Theme "Supercomputing and Modeling for the Human Brain", the Excellence Initiative of the
+German federal and state governments, the Priority Program (SPP 2041 "Computational Connectomics") of the Deutsche
+Forschungsgemeinschaft [S.J. van Albada: AL 2041/1-1], the Helmholtz young investigator's group VH-NG-1028 "Theory of
+multi-scale neuronal networks", and compute time provided by UNINETT Sigma2 - the National Infrastructure for High
+Performance Computing and Data Storage in Norway and its predecessors.
+
+.. image:: _static/img/HBP.png
+  :width: 45 %
+.. image:: _static/img/EBRAINS.png
+  :width: 45 %
```

Comment:
Looks good. But the ebrains logo has a light grey background, does that have to be?

Reply:
On GitHub the ebrains Logo is much bigger than the HBP one. 
https://github.com/nest/nest-simulator/blob/d348fe0f42c2ca38e557fda05be83195ecfc7fad/doc/index.rst
Could that still be adapted?

Reply:
Would suggest not to optimise for rendering on GitHub as (per earlier discussions) their rendering engine is terrible. Importantly, it should look good on ReadTheDocs.

Reply:
> Would suggest not to optimise for rendering on GitHub as (per earlier discussions) their rendering engine is terrible. Importantly, it should look good on ReadTheDocs.

I agree that it should render well on Read the Docs.

> Looks good. But the ebrains logo has a light grey background, does that have to be?

Ideally not. If we could find an EBRAINS PNG that is transparent, like the HBP one, that would be optimal. This is how the EBRAINS logo appears at the moment:

![Screenshot from 2020-06-15 13-37-55](https://user-images.githubusercontent.com/58212853/84653392-9f48b780-af0d-11ea-861a-7cdfd57261d1.png)